_posts/2015-09-17-Rust-1.3.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+title: "Announcing Rust 1.3"
+author: The Rust Core Team
+---
+
+The gear keeps turning: we're releasing Rust 1.3 stable today! As always, read
```

Comment:
if only we had some tooling to statically verify that a reference points to the correct referent
![trollface](https://cloud.githubusercontent.com/assets/27786/9940093/aa82e2ca-5d3b-11e5-9459-9cd5654443d1.png)